### ISSUE TITLE
[SCREENREADER]: Alt attributes need to be more descriptive

### ISSUE THREAD

**Issue by TiffanyPender** [department-of-veterans-affairs/va.gov-team#43898]
-[508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Note: This is about the Veteran-portrait hero area, which is a fall back display that only appears when benefit promo is not present.**

## Feedback framework

- **✔️ Consider** for suggestions/enhancements - nice for screenreader users to hear more descriptive detail about the images on the page

## Definition of done

1. Review and acknowledge feedback.
2. Fix and/or document decisions made.
3. Acceptance criteria listed below met.
4. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## User Story or Problem Statement

As a screenreader user, I expect the alt text used for images to include descriptive text about the image, so that I understand the context and meaning of the image.

## Details

The three alternating images used in the hero content area `data-widget-type="homepage-hero-randomizer"` are not very descriptive. For screenreader users, the alt attribute text should use more descriptive text.

## Acceptance Criteria

- [ ] Alt attribute text for the 3 alternating/randomized images in the hero content area is descriptive

## Environment

* macOS Monterey, `<Version 12.4>`
* Chrome, `<Version 102.0.5005.61>`
* VoiceOver
* Staging

## Steps to Recreate

1. Enter `https://staging.va.gov/homepage-test/` in browser address bar
2. Start screenreading device listed in Environment and listen to the alt text descriptions of the 3 randomized images in the hero content area
3. Verify that the text read is not very descriptive

## Proposed Solution (if known)

Include more descriptive alt text for the hero content images. Possibly consider adding information describing what the person is doing in the photo, race/ethnicity, gender, or other trait, what they are wearing, or maybe branch of military served.

I’ve included suggestions below for the current images on staging, but can draft additional suggestions for any others. The text below should be updated to reflect the correct information, such as first names and service branch, etc.

*Suggestions:*
* Headshot of ##### Babu, ##### male, ##### veteran, wearing ##### hat, jacket thrown over shoulder
* Headshot of ##### Packard-Locke, Caucasian male, ##### veteran, wearing a ball-cap and t-shirt, being hugged and kissed by #####
* Headshot of ##### Britton, African-American male, Marine veteran, wearing a Montford Point ball-cap and ##### windbreaker
* Headshot of ##### Clark, ##### male, ##### veteran, wearing a patriotic shirt
* Headshot of ##### Henline, ##### male, ##### veteran, wearing a tank top, holding a curly haired dog
* Headshot of ##### Dinh, AAPI #####, ##### veteran, wearing a pilot suit, arms crossed
* Headshot of ##### Singleton, ##### woman, ##### veteran, wearing ##### uniform
* Headshot of ##### Cogswell, ##### #####, ##### veteran, wearing ##### uniform

## WCAG or Vendor Guidance (optional)

* [How to design great alt text](https://www.deque.com/blog/great-alt-text-introduction/)

## Screenshots or Trace Logs

![VA home page veterans - Babu, Pickard-Locke, and Britton](https://images.zenhubusercontent.com/62bf0d8f9f16fc23fb806e0a/97c205ec-b245-49d4-ae34-ecf8a3e77e6a)
![VA home page veterans - Clark, Henline, and Dinh](https://images.zenhubusercontent.com/62bf0d8f9f16fc23fb806e0a/b1c99beb-309a-4e38-8884-5feb01281161)
![VA home page veterans - Singleton and Cogswell](https://images.zenhubusercontent.com/62bf0d8f9f16fc23fb806e0a/57217a59-8b56-45dc-a809-9593cb1f0515)

**Comment by laflannery** [comment]
These are decorative images so they should have a null alt when added to the homepage.  This will be handled in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12292)

**Comment by jilladams** [comment]
Got confirmation in Slack: https://dsva.slack.com/archives/C52CL1PKQ/p1686239827957509?thread_ts=1683140159.579789&cid=C52CL1PKQ

* Round 4 version here is final: https://www.sketch.com/s/3aa40506-4be2-46cc-876b-93f1a9f3a857/p/2469A44A-7547-4801-BC97-B70864FBD231/canvas
* Authed version: 

![image (20)](https://github.com/department-of-veterans-affairs/va.gov-team/assets/85581471/f7e7580e-58f1-4c63-8003-0e230fbe2055)
* Per DaveC in slack: 
    * Yes we know it looks weird
    * We dont know how many people are returning to the homepage in the Auth state, so we'll want to check analytics on that.
    * Yes Dave already has ideas about how we might use that empty space if, in fact, this is a problem to be solved b/c  the data supports that Auth users care about the homepage after they signed in.

**Comment by jilladams** [comment]
@laflannery the null alt has shipped: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12292 -- ok to close this ticket?

**Comment by laflannery** [comment]
@jilladams Yep, closing!
